packages/experiments/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -98,12 +100,18 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
         // tslint:disable-next-line:jsx-no-lambda
         onRenderData={ (data: ICommandBarData) => {
           return (
-            <div className={ css(this._classNames.root) }>
-
+            <FocusZone
+              className={ css(this._classNames.root) }
+              direction={ FocusZoneDirection.horizontal }
+              role={ 'menubar' }
+              aria-label={ ariaLabel }
```

Comment:
That was my original approach but @cschlechty advised using aria-label instead of aria-describedby. When I tried it out, the screen reader behavior was to read the command bar aria label immediately after the aria label of the first command in the command bar (ie "New. Use the left and right arrow keys to navigate and the Enter key to activate.")